sphinx/builders/html/__init__.py

```diff
@@ -271,6 +272,26 @@ def init_highlighter(self) -> None:
             style = 'sphinx'
         self.highlighter = PygmentsBridge('html', style)
 
+        if self.config.html_pygments_dark_style is not None:
```

Comment:
Note: We use `html_theme_options` to override the options of HTML themes. So I thought `html_theme_options['pygments_dark_style']` would be better than `html_pygments_dark_style`. But we also use `pygments_style` to override the default style of pygments. It's a common option for all output formats, not only for HTML. I can't say which is better for this case. Either is fine to me.

Reply:
I will leave it at `html_pygments_dark_style` for now though I am open for feedback if someone has some advantages/disadvantages

Reply:
Removed the option totally for now